docs/guides/jobs/tensorboard.md

```diff
@@ -86,12 +85,9 @@ to stop it explicitly. It will be up until your job finishes and then stop autom
 Tensorboard will become inaccessible when the job finishes in any of the `Success`, `Failed`,
 `Timeout` or `Shutdown` states.
 
-### Tensorboard in Jupyter mode
+### Tensorboard with Workspaces
 
-Tensorboard can be run in Jupyter mode as well. You will notice that the links for both
-the Jupyter notebook and the Tensorboard appear in the Job page.
-
-![Tensorboard and Jupyter](../../img/tensorboard_jupyter.png)
+Tensorboard can be run with Workspaces. Open the Tensorboard link the bottom navbar next to the System Metrics to view Tensorboard from your running Workspace.
```

Comment:
This should be rephrased as "Tensorboard is always run inside workspaces. Open the Tensorboard link .."